CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+# Contributing
+
+Thank you for your interest in contributing! We welcome all contributions no matter their size. Please read this guide to learn how to get started. More information about the individual sub-repos can be found in the relevant README.md, as well as on the [airswap docs site](https://docs.airswap.io/).
+
+## Set up:
+First fork airswap-protocols repository and then clone the project. 
+
+Example of clone command after forking with https:
+
+`git clone https://github.com/<YOUR-GITHUB-USER>/airswap-protocols`
+
+This project has multiple sub-repos found within source and utils. We use lerna to handle managing the mono-repo. [Lerna](https://lerna.js.org/) is a tool for managing JavaScript projects with multiple packages.
+
+Run the below command from the root directory to start downloading necessary packages:
+
+`yarn install`
+
+
+## Running the tests
+A great way to explore the code base is to run the tests.
+
+We can run all tests from the root with:
+
+`yarn test`
+
+
+If you want to run the tests of just 1 sub-repo, then run the same command inside the sub-repo.
+
+```
+cd source/swap
+yarn test
+```
+
+## Code Style
+When multiple people are working on the same body of code, it is important that everyone conforms to a similar style. It often doesn’t matter as much which style, but rather that they conform to one style.
+
+To ensure your contribution conforms to the style being used in this project, we require that a linter is run prior to committing to ensure styling. We have also documented our (Solidity style guide)[SOLIDITY_STYLE_GUIDE.md] in the repo as well.
```

Comment:
Incorrect bracketing types on `(Solidity style guide)[SOLIDITY_STYLE_GUIDE.md]`, that won't be a hyperlink. You need [] around the text and then () around the link